tests/common.bash

```diff
@@ -80,10 +80,29 @@ function handle_error() {
 }
 trap 'handle_error $LINENO' ERR
 
+# A wrapper function for kubectl with retry logic
+# runs the command up to 3 times with a 5-second interval
+# to ensure successful execution
+function kubectl_retry() {
+	local max_tries=3
+	local interval=5
+	local i=0
+	while [ $i -lt $max_tries ]; do
+		kubectl $@ && return 0 || true
+		i=$((i + 1))
+		[ $i -lt $max_tries ] && echo "kubectl $@ failed, retrying in $interval seconds" || break
```

Comment:
With this `[ $i -lt $max_tries ]` here (which is a good idea, saving the last sleep!) I think the loop can be simply `while true; do`

While in here, could you please single-quote `kube $@`? Like `echo "'kubectl $@' failed, retrying in $interval seconds"

Reply:
Yep, I will follow your advice. Thanks!